doc/examples/segmentation/plot_multiotsu.py

```diff
@@ -0,0 +1,62 @@
+"""
+=======================
+Multi-Otsu Thresholding
+=======================
+
+The multi-Otsu threshold [1]_ is a thresholding algorithm that is used to separate
+the pixels of an input image into several different classes, each one obtained
+according to the intensity of the gray levels within the image.
+
+Multi-Otsu calculates several thresholds, determined by the number of desired
+classes. The default number of classes is 3: for obtaining three classes, the
+algorithm returns two threshold values. They are represented by a red line in
+the histogram below.
+
+.. [1] Liao, P-S., Chen, T-S. and Chung, P-C., "A fast algorithm for multilevel
+       thresholding", Journal of Information Science and Engineering 17 (5):
+       713-727, 2001. Available at:
+       <http://ftp.iis.sinica.edu.tw/JISE/2001/200109_01.pdf>.
+"""
+
+import matplotlib
+import matplotlib.pyplot as plt
+import numpy as np
+
+from skimage import data
+from skimage.filters import threshold_multiotsu
+
+# Setting the font size for all plots.
+matplotlib.rcParams['font.size'] = 9
+
+# The input image.
+image = data.camera()
+
+# Applying multi-Otsu threshold for the default value, generating
+# three classes.
+thresholds = threshold_multiotsu(image)
+
+# Using the values on thresh, we generate the three regions.
```

Comment:
Using the threshold values

Reply:
@seberg helped with that Emma! But without your work we couldn't do it :smile: Thank you very much!